WordPress/src/main/java/org/wordpress/android/ui/ActivityLauncher.java

```diff
@@ -159,6 +161,15 @@ public static void viewCurrentBlogPlugins(Context context, SiteModel site) {
         }
     }
 
+    public static void viewPluginDetail(Context context, SiteModel site, PluginModel plugin) {
```

Comment:
I'm worried there might be circumstances where these fields are `null`. If that's not possible 👍  if it is could we verify and `return;` early here?

Reply:
I'd avoid doing that for consistency, since every other `ActivityLauncher` method follows the same pattern.